contrib/natss/config/200-dispatcher-service.yaml

```diff
@@ -18,13 +18,14 @@ metadata:
   name: natss-ch-dispatcher
   namespace: knative-eventing
   labels:
-      messaging.knative.dev/channel: natss-channel
-      messaging.knative.dev/role: dispatcher
+    eventing.knative.dev/release: devel
+    messaging.knative.dev/channel: natss-channel
+    messaging.knative.dev/role: dispatcher
 spec:
   selector:
-      messaging.knative.dev/channel: natss-channel
-      messaging.knative.dev/role: dispatcher
+    messaging.knative.dev/channel: natss-channel
+    messaging.knative.dev/role: dispatcher
   ports:
-    - port: 80
```

Comment:
FWIW, my preference is to indent the list items.

Reply:
Either works for me, as long as we're consistent. I thought we were using this style consistently but now I see a lot of variation. I'll create an issue to clean these up.